vaadin-grid-flow-integration-tests/benchmark/run-benchmark.js

```diff
@@ -0,0 +1,195 @@
+const { spawn, execSync } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+const GRID_DIR = '../../';
+const REF_GRID_DIR = './reference-clone';
+const TEST_DIR = 'vaadin-grid-flow-integration-tests';
+
+const gridPath = path.resolve(GRID_DIR);
+const refGridPath = path.resolve(REF_GRID_DIR);
+
+const gridTestPath = path.resolve(gridPath, TEST_DIR);
+const refGridTestPath = path.resolve(refGridPath, TEST_DIR);
+
+const resultsPath = path.resolve('./results');
+
+const REF_JETTY_PORT = 8088;
+// TODO: Fix as a Grid tag of a revision that will be used as the baseline/reference for the tests
+const REF_GIT_TAG = 'benchmark';
```

Comment:
What's the plan for this TODO?
Should the value be passed as a command-line argument?
Are you planning to resolve it within the scope of this PR?

Reply:
The plan was to have it updated once the PR was merged. But I guess we can just update the "benchmark" branch instead and not use a tag for this. Removed the TODO.

Reply:
Ok, fine for me either way 👍